urth_dash_js/notebook/deploy-menu/deploy-menu.js

```diff
@@ -80,5 +80,25 @@ define([
             .on('click', function() {
                 do_bundle('zip');
             });
+        $li = $('<li>')
+        .insertAfter($('#download_ipynb'));
+        $('<a>')
+        .text('iPython Bundled Notebook (.zip)')
+        .appendTo($li)
+        .on('click', function() {
```

Comment:
Can you use the `do_bundle` function from the top of the same JS file? If not, what's required so that we can. It handles save before download.